test/Tester/HostedTestClusterBase.cs

```diff
@@ -14,7 +14,7 @@ public abstract class HostedTestClusterEnsureDefaultStarted : OrleansTestingBase
     {
         protected TestCluster HostedCluster { get; private set; }
 
-        public HostedTestClusterEnsureDefaultStarted(DefaultClusterFixture fixture)
+        public HostedTestClusterEnsureDefaultStarted(BaseTestClusterFixture fixture)
```

Comment:
this base class can only be used with the DefaultClusterFixture, as it uses an collection fixture. If you need a non-default cluster, start a new one using IClassFixture. Look at `GrainCancellationTokenTests` for an example, where it doesn't use this base class at all.